### PR TITLE
Refactor form tests for jsonapi

### PR DESCRIPTION
**Pull request by paulfalgout**
Shortcut Story ID: [sc-45142]

There's a reasonable chance something is failing somewhere, but I'm pushing for 👀 rather than sitting on it.

### CODE REVIEW

File: test/README.md
Comment (nmajor25) [inline test/README.md:1]: Not a big deal, but we could update the `cy.route` from this example code as well: https://github.com/RoundingWell/care-ops-frontend/blob/develop/test/README.md?plain=1#L62

File: test/integration/forms/action.js [hunk 1]
```diff
@@ -4,34 +4,46 @@ import dayjs from 'dayjs';
...
 
 context('Patient Action Form', function() {
   beforeEach(function() {
     cy.routesForDefault();
   });
 
   specify('deleted action', function() {
+    const errors = getErrors({
+      status: '410',
+      title: 'Not Founddddd',
```
Comment (nmajor25) [inline test/integration/forms/action.js:31]: `Not Founddddd`? Was that intentional?

**Review by nmajor25 — APPROVED** [review]

File: test/integration/forms/action.js (hunk 1 above)
Comment (paulfalgout) [reply test/integration/forms/action.js:31]: ha.. kind of.   I initially made it the same as what we were getting back, but when I would flip between 404 and 410 for testing I wasn't seeing any change to what the endpoint was returning.  So I made it obviously different, but still wasn't seeing the change..  So I wasn't sure if I was somehow hitting my docker or what was going on which led to https://github.com/RoundingWell/care-ops-frontend/pull/1200/commits/e27b65d1e18d18b6dcb51c16c4eabb710be51282  I guess I could remove that, but honestly that text doesn't matter.  If we ever need to support multiple languages that value would be returned in the language of the user so we shouldn't use it as a value to be checked against.

File: test/README.md
Comment (paulfalgout) [reply test/README.md:1]: Yeah might as well, though really the whole thing needs new eyes https://app.shortcut.com/roundingwell/story/30274/test-docs-are-outdated

File: test/integration/forms/action.js (hunk 1 above)
Comment (nmajor25) [reply test/integration/forms/action.js:31]: Gotcha, makes sense 👍

Comment (shadowhand) [reply test/integration/forms/action.js:31]: This should really be `Gone` because that's the description of HTTP 410.

**Pull request merged**